go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/ipfs/go-datastore v0.4.4
 	github.com/ipfs/go-log v1.0.4
 	github.com/keep-network/go-libp2p-bootstrap v0.0.0-20200423153828-ed815bc50aec
-	github.com/keep-network/keep-common v1.1.1-0.20200805115808-67d9cb633ddb
+	github.com/keep-network/keep-common v1.1.1-0.20200819140235-09eff2362ac0
```

Comment:
We have to update this one after merging https://github.com/keep-network/keep-core/pull/1959.